server/middleware/case.js

```diff
@@ -91,6 +93,58 @@ function caseCorrespondentsApiResponseMiddleware(req, res) {
     return res.status(200).json(res.locals.correspondents);
 }
 
+async function caseActionDataMiddleware(req, res, next) {
+    try {
+        const preppedData = {};
+        const actionData = await req.listService.fetch('CASE_ACTIONS', req.params);
+        let uniqueActionTypeIds;
+        let uniqueActionTypes;
+
+        if (actionData && actionData.caseTypeActionData) {
```

Comment:
These first two ifs both have the same condition `(actionData && actionData.caseTypeActionData)`.

If both of these variables must be defined for the successful execution of the function, could we check for their presence at the start, then throw and log a meaningful error if they're missing? We could then get rid of the ifs.

Reply:
I'll alter to combine the IF statements... it needs to be able to successfully return even if there is no caseActionData, I see no need to throw. I'll add a log message at warn level to advise no info found.